database/index.js

```diff
@@ -89,8 +89,23 @@ const insertIntoMatProd = (dataSet) => {
   return Promise.all(promiseArray);
 }
 
+const getProducts = () => {
```

Comment:
@gigicupp Have you tested this function? Does the returned promise contain the data that you want it to contain? If it does, great!

Reply:
@masoncpott yep, it contains all the details from the table.